tests/app.test.ts

```diff
@@ -1,8 +1,99 @@
 import request from "supertest";
-import  app from "../source/app";
+import router from '../routes/routes';
+import { Soldier } from "../repository/repo";
+const app = router; 
+beforeEach(async ()=>{ 
+    await Soldier.deleteMany();
+});
 
 describe('Is Soldier Alive', () => {
```

Comment:
Soldier isn't alive, the server might be alive, place this test in a separate place - this is a test for the health route.

The soldier route should have a separate test file